Snakefile-sorting-hat

```diff
@@ -165,7 +165,7 @@ rule get_experiment_metadata:
         export ATLAS_META_CONFIG={params.atlas_meta_config}
         export ATLAS_PROD={params.atlas_prod}
         export ATLAS_EXPS={params.atlas_exps}
-        source {workflow.basedir}/bin/metadata_retrieval_functions.sh
+        source {workflow.basedir}/atlas-bash-util/generic_routines.sh
```

Comment:
My concern here is that metadata_retrieval_functions was very specific and generic routines is quite wide? Is this replacement working well?

Reply:
I have checked that all the code being replaced is equal. It works